src/System.Text.Json/src/System/Text/Json/Serialization/JsonPropertyNameAttribute.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.Json.Serialization
+{
+    /// <summary>
+    /// Specifies the property name that is present in the JSON when serializing and deserializing.
```

Comment:
`present in the JSON when serializing` doesn't make much sense. This attribute likes on the POCO, right, and the JSON being written from the POCO uses it.